visualizer/bytesprites/bytesprite.py

```diff
@@ -9,10 +9,92 @@
+            The active_sheet is the list of images (sprites) that is currently being used. In other words, it's a strip
+            of sprites that will be used.
+
+        Spritesheets:
+            This is a 2D array of sprites. For example, refer to the ``ExampleSpritesheet.png`` file. The entirety of the
+            4x4 would be a spritesheet. One row of it would be used as an active sheet.
+
+        Object Type:
+            This is an int that represents the enum value of the Object the sprite represents. For example, the
+            ``ExampleSpritesheet.png`` shows the Avatar. The Avatar object_type's enum value is found in the JSON logs and
+            is the number 4. This would change if the order of the ObjectType enum changes, so be mindful of that and
+            refer to the JSON logs for the exact values.
+
+        Rect:
+            The rect is an object used for rectangular objects. You can offset the top left corner of the Rect by
+            passing parameters.
+
+            Example:
+
+            On the left, the Rect's offset is depicted as being at (0, 0), meaning there is no offset. On the right,
+            the Rect is seen further to the right, showing its offset from the left corner of the screen.
+
+        Rect Example:
+        ::
+            -----------------------                 -----------------------
+            |------               |                 |     ------          |
+            ||    |               |                 |     |    |          |
+            |______               |    -------->    |     ______          |
+            |                     |                 |                     |
+            |                     |                 |                     |
+            |                     |                 |                     |
+            -----------------------                 -----------------------
+
+        Screen:
+            The screen also a PyGame.Screen object, so it simply represents an image of the screen itself.
+
+        Image:
+            The image is an individual sprite in a spritesheet.
+
+        Frame Index:
+            The frame index is an int that is used to determine which sprite to use from the active_sheet. For example,
+            say the active_sheet is the first row in the ``ExampleSpritesheet.png``. If the frame_index is 1, the first
+            image will be used where the head is centered. If the frame_index is 3, the sprite used will be where the
+            head off to the right.
+
+        Config:
+            This is an object reference to the ``config.py`` file. It's used to access the fixed values that are only
+            accessed in the configurations of the file.
+
+        Update Function:
+            The update function is a method that is assigned during instantiation of the Bytesprite. That function is
+            used to update which active_sheet is used depending on what is implemented in Bytesprite classes.
```

Comment:
should you use "whichever" rather than "which" -OR- use "is being used" rather than "is used"?